astropy_timeseries/io/core.py

```diff
@@ -0,0 +1,109 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
I think the name of this file is too generic, as we might add formats not related to Kepler or TESS. I think since the Kepler format came first and TESS then adapted it, it would make sense to just call this ``kepler.py`` and then add a note that this also deals with the case of TESS.

Reply:
My original idea and probably in hindsight a stupid one, was that there would be one general IO function that would be used to read all specific files and inside it would call out to instrument specific functions. Which was the basis for the `_unspported` function.